README.md

```diff
@@ -1 +1,80 @@
-# achinn
\ No newline at end of file
+# achinn
+
+This is a living document about me.
+
+It provides a way for others to quickly get up to speed with how I work.
+
+It provides a way for me to build my own self awareness.
+
+
+## Public endpoints
+
+### `/slack`
+
+If you need something immediate from me, this or [`/ad-hoc-irl`](#ad-hoc-irl) are the endpoints to use. For the most part I respond quickly but [there are](#context-switching) [instances](#meetings) where I may not. In those cases I will leave your message unread and circle back later. If I'm offline, however, all bets are off.
```

Comment:
`If I'm offline, however, all bets are off` Does this mean that you might not respond?

Reply:
```suggestion
If you need something immediate from me, this or [`/ad-hoc-irl`](#ad-hoc-irl) are the endpoints to use. For the most part I respond quickly but [there are](#context-switching) [instances](#meetings) where I may not. In those cases I will leave your message unread and circle back later. If I'm offline, however, don't expect an immediate response.
```